### ISSUE TITLE
dupe in grid

### ISSUE THREAD

**Issue by CarlFK** [pydanny/djangopackages#321]
https://www.djangopackages.com/grids/g/countries/

django-airports 	django-airports 

Not sure if this is a data problem or a code problem.

**Comment by pydanny** [comment]
A bug of some sort. We'll look into it!

**Comment by benebun** [comment]
FYI can't reproduce this atm, temporary glitch/caching?

**Comment by pydanny** [comment]
It's actually the same package added twice. Trying to figure out how that happened...

**Comment by radiac** [comment]
Sorry, I've just done this too, with django-yarr on https://www.djangopackages.com/grids/g/feed-aggregators/

Here's how I managed it:

1. Go to package page
2. Click on ``+`` next to ``Comparison Grids``
3. Type in the grid to add it to
4. Follow the instructions to ``Click a grid`` by clicking on the grid
5. Notice the grid is now listed twice, assume it's your fault, and spend half an hour trying to find a way to fix your mistake before anyone notices ;)

I tracked down the problem to there being two ``form#find-a-grid-form`` on the page, so the javascript is submitting both - and I guess they're submitted so close together that they're finding their way through the race condition in ``add_grid_package``.

I'll fix this in the templates in a pull request shortly, but would it also be an idea to add a ``unique_together`` constraint to the ``GridPackage`` model so it can't happen if the js breaks in the future?
